.github/workflows/ci.yml

```diff
@@ -24,24 +24,25 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Build the image
-        run: "cekit --verbose build --overrides \"{'labels': [{'name': 'quay.expires-after', 'value': '1w'}, {'name': 'git-sha', 'value': '$GITHUB_SHA'}]}\" docker --tag $IMAGE_NAME:latest"
+        run: >
+          cekit --verbose build --overrides "{'labels': [{'name': 'quay.expires-after', 'value': '1w'}, {'name': 'git-sha', 'value': '$GITHUB_SHA'}]}" docker --tag $IMAGE_NAME:latest
```

Comment:
Remains docker, `cekit build podman`

Reply:
And you will need to add a source for get podman on ubuntu. I still not much understand why we are replacing docker cli on ubuntu machine in GH Actions with docker but anyway :sweat_smile: 

```
echo "deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_${VERSION_ID}/ /" | sudo tee /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
curl -L https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_${VERSION_ID}/Release.key | sudo apt-key add -
sudo apt-get update
sudo apt-get -y upgrade 
sudo apt-get -y install podman
```

Reply:
Podman is already available on GitHub Actions Containers: https://github.com/actions/virtual-environments/issues/320, we don't need to install it 😅 🎉

Reply:
I have pushed another commit to remove the latest references to docker